src/base/bittorrent/session.cpp

```diff
@@ -1444,12 +1444,16 @@ void Session::configureNetworkInterfaces(lt::settings_pack &settingsPack)
         const QHostAddress addr {ip};
         if (!addr.isNull())
         {
-            const QString ip = ((addr.protocol() == QAbstractSocket::IPv6Protocol)
-                          ? ('[' + Utils::Net::canonicalIPv6Addr(addr).toString() + ']')
+            const bool isIPv6 = addr.protocol() == QAbstractSocket::IPv6Protocol;
```

Comment:
Some clarifications:
I prefer the version without curly brackets. I think it better fits the style of previous coding.